<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinBox_2">
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="value">
          <number>166</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Largeur :</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Hauteur :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinBox">
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="value">
          <number>80</number>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Enregistrer</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Recalculer</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Mailles :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Type :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>TYPE I</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TYPE II</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TYPE III</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TYPE IV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TYPE V</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TYPE VI</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>589</width>
          <height>248</height>
         </rect>
        </property>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>8</x>
           <y>4</y>
           <width>571</width>
           <height>281</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
